fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -597,12 +618,16 @@ class ShardManager {
 	}
 
 	rocksdb::DB* getDb() { return db; }
+
 	std::unordered_map<std::string, std::shared_ptr<PhysicalShard>>* getAllShards() { return &physicalShards; }
 
+	std::unordered_map<uint32_t, rocksdb::ColumnFamilyHandle*>* getColumnFamilyMap() { return &columnFamilyMap; }
+
 private:
 	std::string path;
 	rocksdb::DB* db = nullptr;
 	std::unordered_map<std::string, std::shared_ptr<PhysicalShard>> physicalShards;
+	std::unordered_map<uint32_t, rocksdb::ColumnFamilyHandle*> columnFamilyMap;
```

Comment:
Please add a comment about the meaning of columnFamilyMap.

In addition, could `columnFamilyMap` be a member of the `Reader` and `Writer` structs, if it is a for the entire kv?

Reply:
Moved it to writer.